app/src/main/res/values/strings.xml

```diff
@@ -61,6 +61,7 @@
     <string name="about_license">License</string>
     <string name="about_questionnaire">Questionnaire</string>
     <string name="about_version_prefix">"V%1$s</string>
+    <string name="about_site_url">https://droidkaigi.github.io/2016/en</string>
```

Comment:
One suggestion.
I guess it is better to create `values-ar/strings_urls.xml` and `values-ja/strings_urls.xml` and `write about_site_url` in that new file. How do you think?